src/practicalities.md

```diff
@@ -4,7 +4,7 @@ page_class: practicalities
 title: "Practicalities"
 ---
 
-On this page you'll find our __A-Z of Practicalities for RubyConf TH 2022__.
+On this page you'll find our __A-Z of Practicalities for RubyConf TH 2023__.
```

Comment:
Maybe let's add a note here like 

`Some of the content on this page was originally written for our 2022 conference, so expect some changes once we have confirmed final details and venue for 2023.`

Reply:
I have added a note and a timestamp:

![image](https://user-images.githubusercontent.com/696529/213647203-c7f542cb-49ff-4ca4-8abd-98d5dd04dccb.png)

I left out the part that the document was created for the other event as I have already edited the content to fir the 2023 event.